### PR TITLE
CDAP-13974 plugin requirement docs

### PR DESCRIPTION
**Pull request by rohitsinha54**
Quick build: https://builds.cask.co/browse/CDAP-DQB434-1

### CODE REVIEW

**Review by albertshau — APPROVED** [review]
lgtm

File: cdap-docs/developer-manual/source/pipelines/developing-plugins/plugin-basics.rst [hunk 1]
```diff
@@ -74,9 +74,7 @@ These annotations are used for plugin classes:
 
 - ``@Description``: Annotation used to add a description of the plugin.
 
-- ``@Requirements``: Annotation used to specify the
-`Requirements <../../reference-manual/javadocs/co/cask/cdap/api/annotation/Requirements.html>`__ needed by a plugin
-to run successfully.
+- ``@Requirements``: Annotation used to specify the `Requirements <../../reference-manual/javadocs/co/cask/cdap/api/annotation/Requirements.html>`__ needed by a plugin to run successfully.
```
Comment (albertshau) [inline cdap-docs/developer-manual/source/pipelines/developing-plugins/plugin-basics.rst]: it works if you split it across lines like this?

**Review by albertshau — APPROVED** [review]
lgtm assuming it builds

File: cdap-docs/developer-manual/source/pipelines/developing-plugins/plugin-basics.rst (hunk 1 above)
Comment (rohitsinha54) [reply cdap-docs/developer-manual/source/pipelines/developing-plugins/plugin-basics.rst]: Yes it does.

https://builds.cask.co/artifact/CDAP-DQB434/shared/build-3/Docs-HTML/5.1.0-SNAPSHOT/en/developer-manual/pipelines/developing-plugins/plugin-basics.html#class-annotations

**Pull request merged**